src/crypto/crypto_random.cc

```diff
@@ -176,6 +176,11 @@ Maybe<void> CheckPrimeTraits::AdditionalConfig(
   ArrayBufferOrViewContents<unsigned char> candidate(args[offset]);
 
   params->candidate = BignumPointer(candidate.data(), candidate.size());
+  if (params->candidate.get() == nullptr) {
```

Comment:
A simplified bool check should work here

```suggestion
  if (!params->candidate) {
```

Reply:
Hadn't noticed the `bool` operator was implemented. Thanks!